qa/tasks/cephfs/mount.py

```diff
@@ -1563,7 +1564,11 @@ def setfattr(self, path, key, val, **kwargs):
         :param val: xattr value
         :return: None
         """
-        kwargs['args'] = ["setfattr", "-n", key, "-v", val, path]
+        if os.uname().sysname == 'Darwin':
```

Comment:
OK thanks for the suggestion I'll investigate this option. These changes are indeed to make the test runnable locally on a mac.

Reply:
You are right, I have not considered these as truly remote commands. However, I also don't fully agree with adding `setfattr` to the bin directory. The thing is, when added in the bin dir, I'd need to take the responsibility to create a fully functional wrapper for how setfattr behaves on a linux system because I'd lose the context of the code that constructed a particular command. That's a little too much IMO.

Currently, I believe the best way to fix the problem you've uncovered is to check the remote sysname rather than the local one

Reply:
I'm moving this code to where it belongs: my [M1 mac support PR](https://github.com/ceph/ceph/pull/52629), where I'll then address this problem.